app/src/main/java/com/nononsenseapps/feeder/ui/compose/deletefeed/DeleteFeedScreen.kt

```diff
@@ -86,14 +107,46 @@ fun DeleteFeedDialog(
             }
         },
         title = {
-            Text(
-                text = stringResource(id = R.string.delete_feed),
-                style = MaterialTheme.typography.titleLarge,
-                textAlign = TextAlign.Center,
-                modifier =
-                    Modifier
-                        .padding(vertical = 8.dp),
-            )
+            Box(modifier = Modifier.fillMaxWidth()) {
+                val stateLabel = if (selectAllSelected) {
+                    stringResource(androidx.compose.ui.R.string.selected)
+                } else {
+                    stringResource(androidx.compose.ui.R.string.not_selected)
+                }
+                Text(
+                    text = stringResource(id = R.string.delete_feed),
+                    style = MaterialTheme.typography.titleMedium,
+                    modifier = Modifier.align(Alignment.CenterStart)
+                )
+                Box(modifier = Modifier.align(Alignment.CenterEnd)) {
+                    IconButton(onClick = { menuExpanded = true }) {
+                        Icon(Icons.Default.MoreVert, contentDescription = stringResource(android.R.string.selectAll))
+                    }
+
+                    DropdownMenu(expanded = menuExpanded, onDismissRequest = { menuExpanded = false }) {
+                        DropdownMenuItem(text = {
+                            Row(verticalAlignment = Alignment.CenterVertically) {
+                                Checkbox(checked = isSelectAllChecked(), onCheckedChange = { checked ->
```

Comment:
```suggestion
                                Checkbox(checked = isSelectAllChecked, onCheckedChange = { checked ->
```

Don't execute the lambda

Reply:
`Checked` here expects a Boolean, How would you prefer this?

Reply:
Doh..

Well, in that case it can't be a lambda. That won't work very well I think. Needs to be a state.